zipkin-storage/dse5/src/main/resources/dse5-schema.cql

```diff
@@ -0,0 +1,70 @@
+CREATE KEYSPACE IF NOT EXISTS zipkin3 WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '1'};
+
+CREATE TYPE IF NOT EXISTS zipkin3.endpoint (
+    service_name text,
+    ipv4         inet,
+    ipv6         inet,
+    port         smallint,
+);
+
+CREATE TYPE IF NOT EXISTS zipkin3.annotation (
+    ts bigint,
+    v  text,
+    ep frozen<endpoint>
+);
+
+CREATE TYPE IF NOT EXISTS zipkin3.binary_annotation (
+    k  text,
```

Comment:
I don't recall if in Cassandra 3.x whether long column names inflates storage use, but I would encourage not using abbreviations.